src/AuditReader.php

```diff
@@ -64,6 +64,9 @@ class AuditReader
      * Entity cache to prevent circular references.
      *
      * @var array
+     *
+     * @phpstan-template T of object
```

Comment:
A template on a property seems weird, it's on the class or on a method generally.
All the `@phpstan-template T of object` might be moved to the class...